wiki/Help_Centre/en.md

```diff
@@ -687,9 +691,11 @@ If you wish to prevent anyone not on your friendslist from sending you private m
 6. Click the first dropdown box once more and change its value to "Delete message", then click "Add Rule".
 7. You are done! Any attempts the user makes to contact you via forum PMs will now be automatically deleted without warning to you.
 
+On the new website, go to your [account settings](https://osu.ppy.sh/home/account/edit) and simply click _lock private messages from people not on your friends list_ in the Privacy section. Private messages from users in your friendlist will still reach you.
```

Comment:
for interface labels, I would use code blocks as well: `lock private messages from people not on your friends list`